GeoFire.podspec

```diff
@@ -1,15 +1,15 @@
 Pod::Spec.new do |s|
   s.name         = "GeoFire"
-  s.version      = "2.0.0"
+  s.version      = "2.0.1"
   s.summary      = "Realtime location queries with Firebase."
   s.homepage     = "https://github.com/firebase/geofire-objc"
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
   s.author       = "Firebase"
-  s.source       = { :git => "https://github.com/firebase/geofire-objc.git", :tag => 'v2.0.0' }
+  s.source       = { :git => "https://github.com/firebase/geofire-objc.git", :tag => 'v2.0.1' }
   s.source_files = "GeoFire/**/*.{h,m}"
   s.documentation_url   = "https://geofire-ios.firebaseapp.com/docs/"
   s.ios.deployment_target = '7.0'
-  s.ios.dependency  'Firebase/Database', '~> 4.0'
+  s.ios.dependency  'FirebaseCommunity/Database', '~> 0.0.1'
```

Comment:
Can this be 0.0 instead?  FirebaseCommunity will likely go to 0.1.0 soon and it shouldn't impact GeoFire.